wails-frontend/wailsjs/go/models.ts

```diff
@@ -0,0 +1,21 @@
+export namespace walletapp {
+	
+	export class selectFileResult {
```

Comment:
class name shouldn't be capitalized: `SelectFileResult`?

Reply:
nope, its generated code!